CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+
+## Get started
+
+1. Join [Code for PDX Discord](https://discord.gg/FEX9KUMH). If you run into any issues, shoot an email to our CodeForPDX brigade leader, Hugh: Hugh@codeforpdx.org
+2. Post your GitHub name in the [**#github-access-request**](https://discord.com/channels/1068260532806766733/1078124139983945858) channel and we’ll add you to our [GitHub](https://github.com/codeforpdx/PASS). Even if you don't plan or writing code for the project, please have Hugh get you connected to our GitHub. We keep track of technical and non-technical tasks in GitHub Projects.
+3. Introduce yourself and post your email in the [**#pass**](https://discord.com/channels/1068260532806766733/1075285803137257544) channel so we can add you to our [Google Drive](https://drive.google.com/drive/u/0/folders/1zTEd34K7Eg7rvg71zS6Uzbwrsct2Lx9E?ths=true). You can send that info to Flo privately if you're more comfortable with that. 
+4. Our project is broken into three teams: UX/UI, Project Management, and Developers (back and front end). We are currently meeting in separate groups based on team member availability. The development team meets virtually every Tuesday at 7pm via Google Meet.
+5. Reach out to Flo (Project Manager) on slack to introduce yourself and share how you wish to contribute.
+
+
+## Ready to contribute?
+
+1. Get our github project cloned to your local computer in whatever IDE you prefer. If you are new to Git/GitHub, I have links to learn more as you go. You can also check out [this article](https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github) for a broad overview.
+  * Open terminal
+  * Change working directory to the location where you want the cloned directory.
+  * `git clone https://github.com/codeforpdx/PASS.git` [learn more](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository?platform=linux)
+  * `git remote add origin https://github.com/codeforpdx/PASS.git` [learn more](https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories)
+2. Once you have PASS cloned to your local system, read the [React Documentation](/docs/README.md). Follow the instructions to open JSDocs and review Forms, hooks, utils and solid_identity_provider tabs. This will give you a great initial breakdown of the project functions.
+3. Feel free to [`git checkout jk/solidTests`](https://github.com/codeforpdx/PASS/tree/jk/solidTests). If you notice a gap or errors in documentation feel free to make the changes locally and create a pull request to the branch creator.
+
+
+## Our development process
+
+1. Follow step one under **Ready to contribute?**
+2. Create a new branch to work on your feature:
+  * `git checkout -b <branch name>` [learn more](https://www.atlassian.com/git/tutorials/using-branches/git-checkout)
+3. `git pull origin Development` - to sync with PASS Development branch
```

Comment:
Minor edit to the `git checkout` command:

`git checkout -b <your branch name> Development`

This will ensure the new branch is created off of the `Development` branch.